.github/workflows/build.yml

```diff
@@ -102,9 +100,15 @@ jobs:
         shell: bash
         run: |
           tree /tmp/benchmarks
+          mkdir results
+          for el in $(find /tmp/benchmarks -type f); do # cleanup tree for easier reading of report
+            artifact_name=$(basename $(dirname $(dirname $el)))
+            mkdir -p results/$artifact_name
+            mv $el results/${artifact_name}/
+          done
           mkdir html
           echo "$(date)" > raw
-          PY_COLORS=1 py.test-benchmark compare $(find /tmp/benchmarks -type f) --histogram histograms --group-by name --csv results.csv --sort name >> raw
+          PY_COLORS=1 py.test-benchmark compare $(find results/ -type f) --histogram histograms --group-by name --csv results.csv --sort name >> raw
```

Comment:
So the mistake previously was grouping by fullfunc which grouped several sub benchmarks together if they existed in a single test function?

Reply:
Indeed, there are several grouping options available:
```bash
for type in 'group' 'name' 'fullname' 'func' 'fullfunc' 'param'; do
    echo "**** $type ****"
    py.test-benchmark compare $(find results -type f) --sort name --group-by $type
done
```